core/daemon/src/main/java/org/opennms/netmgt/vmmgr/Controller.java

```diff
@@ -426,4 +463,8 @@ public int getRmiHandshakeTimeout() {
     public void setRmiHandshakeTimeout(int httpRequestReadTimeout) {
         System.setProperty("sun.rmi.transport.tcp.handshakeTimeout", String.valueOf(httpRequestReadTimeout));
     }
+
+    static void setSecureCredentialsVault(SecureCredentialsVault secureCredentialsVault) {
```

Comment:
Guess this is only needed for testing purposes.
Better to add `@VisibleForTesting` annotation

Reply:
done.